createdisk-library.sh

```diff
@@ -116,6 +119,10 @@ function update_json_description {
         | ${JQ} '.storage.diskImages[0].format = "qcow2"' \
         | ${JQ} ".storage.diskImages[0].size = \"${diskSize}\"" \
         | ${JQ} ".storage.diskImages[0].sha256sum = \"${diskSha256Sum}\"" \
+        | ${JQ} ".storage.additionalFiles[0].name = \"oc\"" \
+        | ${JQ} '.storage.additionalFiles[0].type = "oc-executable"' \
```

Comment:
@cfergeau do we really need this `oc-executable` here if in the template it is same for all platform?

Reply:
I can see this also being used for `podman`, and the name is platform-dependant (because of the `.exe` extension on Windows), so this 'type' gives us a generic identifier which will be the same on all platforms. Without it we'll have to make some guesses.